File: programmers/기초/문자 개수 세기.js
//문자 개수 세기
function solution(my_string) {
  let alpha = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
  let result = new Array(52).fill(0);

  my_string.split("").map((n) => {
    result[alpha.indexOf(n)] += 1;
  });

  return result;
}

File: programmers/python/문자 개수 세기.py
# 문자 개수 
def solution(my_string):
    answer = []
    alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
    for i in alphabet:
        answer.append(len(my_string.split(i))-1)
    return answer



def solution(my_string):
    answer=[0]*52
    for x in my_string:
        if x.isupper():
            answer[ord(x)-65]+=1
        else:
            answer[ord(x)-71]+=1
    return answer